declarations/WebpackOptions.d.ts

```diff
@@ -1198,6 +1198,10 @@ export interface OutputOptions {
 	 * Module namespace to use when interpolating filename template string for the sources array in a generated SourceMap. Defaults to `output.library` if not set. It's useful for avoiding runtime collisions in sourcemaps from multiple webpack projects built as libraries.
 	 */
 	devtoolNamespace?: string;
+	/**
+	 * The maximum EcmaScript version of the webpack generated code (doesn't include input source code from modules).
+	 */
+	ecmaVersion?: 5 | number;
```

Comment:
eslint supports years also, maybe will be good have this ability too?

Reply:
It's years. The schema is `5 | number > 2015`. ES5 has no year, so this is kind of the exception

Reply:
:+1: